marimo/_cli/envinfo.py

```diff
@@ -0,0 +1,139 @@
+# Copyright 2023 Marimo. All rights reserved.
+from __future__ import annotations
+
+import json
+import platform
+import subprocess
+import sys
+from typing import Optional, Union
+
+from marimo import __version__
+
+
+def get_system_info() -> dict[str, Union[str, dict[str, str]]]:
+    info = {
+        "marimo": __version__,
+        "OS": platform.system(),
+        "OS Version": platform.release(),
+        "Python Version": platform.python_version(),
+    }
+
+    binaries = {
+        "Chrome": _get_chrome_version() or "--",
+        "Node": _get_node_version() or "--",
+    }
+
+    requirements = _get_pip_list()
```

Comment:
This will work for now, but eventually we'll be on `conda`, at which point we might add a flag that controls whether this command looks at `conda` or `pip`. Just pointing it out, no action needed